### PR TITLE
[dox] Update getopt.d

### PR DESCRIPTION
**Pull request by jibal**
Documentation fixes and improvements.

### CODE REVIEW

**Comment by quickfur** [comment]
LGTM

File: std/getopt.d [hunk 1]
```diff
@@ -468,13 +467,13 @@ unittest
...
+    /// Turn case sensitivity on
     caseSensitive,
-    /// Turns case sensitivity off
+    /// Turn case sensitivity off
     caseInsensitive,
-    /// Turns bundling on
+    /// Turn bundling on
     bundling,
-    /// Turns bundling off
+    /// Turn bundling off
```
Comment (burner) [inline std/getopt.d]: why did you remove the "s" if one views the docs.

caseSensitive\n Turns case sen.... 

looks ok

File: std/getopt.d [hunk 2]
```diff
@@ -88,7 +88,7 @@ void main(string[] args)
  The $(D getopt) function takes a reference to the command line
  (as received by $(D main)) as its first argument, and an
  unbounded number of pairs of strings and pointers. Each string is an
- option meant to "fill" the value pointed-to by the pointer to its
+ option meant to "fill" the value pointed to by the pointer to its
```
Comment (schveiguy) [inline std/getopt.d]: Ugh, pointed to by the pointer.

How about `"fill" the value referenced by the pointer to its right`

File: std/getopt.d (hunk 1 above)
Comment (jibal) [inline std/getopt.d]: Because the grammar was inconsistent. I changed
 
Turns / Pass / Signal / Stop / Do not / Makes
to
Turn / Pass / Signal / Stop / Do not / Make

File: std/getopt.d (hunk 2 above)
Comment (jibal) [inline std/getopt.d]: Done.

**Comment by DmitryOlshansky** [comment]
LGTM

**Comment by JakobOvrum** [comment]
The commits should be squashed.

**Comment by jibal** [comment]
I don't know how to do that. I simply clicked the "Improve this page" button on the doc page. I'm not familiar with github (yet).

**Comment by burner** [comment]
you don't need github you need git

* use git to clone your repo
* checkout this branch
* http://gitready.com/advanced/2009/02/10/squashing-commits-with-rebase.html
* push --force the squashed branch
* tell us you did so

**Comment by quickfur** [comment]
ping

**Comment by CyberShadow** [comment]
Should I take this over?

**Comment by JackStouffer** [comment]
@CyberShadow please do. The OP seems MIA.

**Comment by CyberShadow** [comment]
https://github.com/D-Programming-Language/phobos/pull/3729
